Integrations/EDL/EDL_test.py

```diff
@@ -274,3 +276,31 @@ def test_validate_basic_authentication(self):
         assert not validate_basic_authentication(data.get('wrong_length_auth'), username, password)
         assert not validate_basic_authentication(data.get('wrong_credentials_auth'), username, password)
         assert validate_basic_authentication(data.get('right_credentials_auth'), username, password)
+
+    @pytest.mark.ips_to_ranges
+    def test_ips_to_ranges_range(self):
+        from EDL import ips_to_ranges, COLLAPSE_TO_RANGES
+        ip_list = [IPAddress("1.1.1.1"), IPAddress("25.24.23.22"), IPAddress("22.21.20.19"),
+                   IPAddress("1.1.1.2"), IPAddress("1.2.3.4"), IPAddress("1.1.1.3"), IPAddress("2.2.2.2"),
+                   IPAddress("1.2.3.5")]
+
+        ip_range_list = ips_to_ranges(ip_list, COLLAPSE_TO_RANGES)
+        assert "1.1.1.1-1.1.1.3" in ip_range_list
+        assert "1.2.3.4-1.2.3.5" in ip_range_list
+        assert "1.1.1.2" not in ip_range_list
+        assert "2.2.2.2" in ip_range_list
```

Comment:
check that 25.24.23.22 in the results as well

Reply:
done